scripts/bootstrap-rootfs-overlay-hosted-server.sh

```diff
@@ -56,9 +56,10 @@ if [ -e ${output_dir} ]; then
      sudo chgrp -R $(id -g) ${output_dir}
 fi
 
-if [ "${region}" = "eu" ]; then
+# check lowercase or uppercase region
+if [ "${region}" = "eu" || "${region}" = "EU" ]; then
```

Comment:
more general, but I was tempted to write `if [ "${region,,}" = 'eu' ]; then`, but I don't know if it would work on any bash version (on zsh does not).